konveyor/tackle-test-generator-cli
# Issue 44 make augmentation option enabled by default

## Description

1.added relevant tests
2. run it on pdl, with several options. clean issues

Related to # 44

## Type of Change

Please check the types of changes your PR introduces.

- [X ] Bug fix (non-breaking change that fixes an issue)
- [ ] Refactoring (non-breaking code restructuring that preserves functionality)
- [ ] Build-related update (CI workflow, test cases)

## How Has This Been Tested?

1. added dedicated test to run augmentation with different parameters
2. run augmentation with almost every ctd-amplified tests
3. check augmentation results

- [x ] Test A
- [x ] Test B

## Checklist

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing tests pass locally with my changes
- [x ] Any dependent changes have been merged and published in downstream modules
